### ISSUE TITLE
Tiny feature request: draft order

### ISSUE THREAD

**Issue by aws0m** [bafolts/terraforming-mars#3105]
would it maybe be possible to make the order of drafts more clear? As it is now you can see how you are currently going to pass to in draft when you look at your draft choices but once you have picked your card you cant see it anymore. And sometimes for me at least I pick something fast and then during the downtime before next draft round I sometimes forget who im drafting to :p

Maybe you could either keep the text about who your drafting to not disappear when you pick your card.
or/and you could have some kind of arrows after/over the player boxes.
or/and you can use that nice UI from when you do initial draft for the turn order, and then you could efficiently see the whole draft order for like a 4p game for example.

As always you guys are the best!

**Comment by sauloachkar** [comment]
Perfect solution for me would be to keep the text, the cards and the button. That way I can keep looking at both the cards, the text. And additionally I wish I could repick the card, while others have not finished picking theirs.

**Comment by kberg** [comment]
I agree that keeping a relatively permanent visualization of the draft order is worthwhile.
